docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,816 @@
+---
+title: 'The 2021 MUI developer survey: here's what we discovered'
+description: Your feedback helps us to build better products. Here's what we learned about your needs in our annual survey.
+date: 2022-03-07T00:00:00.000Z
+authors: ['danilo-leal', 'samuelsycamore', 'oliviertassinari']
+tags: ['Developer survey']
+---
+
+Keeping up with tradition, a few months ago we opened the 2021 MUI Developer Survey.
+Your feedback helps us to build better products, and we can't thank you enough for being a part of our community.
+1,591 of you responded to this year's survey, and we take your input very seriously.
+Here's what we've learned about your needs after poring over the results.
+
+As with the previous iterations, the survey was divided into three sections: ["Your needs"](#your-needs), ["Your product"](#your-product), and ["About you"](#about-you).
+
+## Your needs
+
+### How would you feel if you could no longer use MUI?
+
+Over 93% of you would be disappointed if you could no longer use MUI (a sum of _"Very disappointed"_ and _"Somewhat disappointed"_).
+That's in keeping with 94% of you who felt the same way when we asked this question in 2019 and 2020. The _"Somewhat disappointed"_ also saw a growth of 9%. These indicators highlight that the evolution of the product that MUI provides might be pointed in the right direction!
```

Comment:
Let's make the takeaway more direct.

```suggestion
That's in keeping with 94% of you who felt the same way when we asked this question in 2019 and 2020. The _"Somewhat disappointed"_ also saw a growth of 9%. These indicators suggest to us that we're on the right track with our products and their continued evolution.
```

Reply:
I feel like this is a misleading characterization of what happened with "Somewhat disappointed". The "growth" of 9% was entirely a shift from "Very disappointed" to "Somewhat disappointed", so that growth isn't a positive indicator yet the sentence that follows it seems to imply that it **is** a positive change.

Reply:
Yeah... I think you're right. Maybe a more accurate interpretation would be, starting from the beginning of the answer:

> Over 93% of you would be disappointed if you could no longer use MUI (a sum of "Very disappointed" and "Somewhat disappointed"). That's in keeping with 94% of you who felt the same way when we asked this question in 2019 and 2020. 
>
> However, in comparison with last year, the 9% growth that the "Somewhat disappointed" option saw was most likely from developers who would've had placed their answer in "Very disappointed", since that option had also a 10% drop. Also, those who would be "Not disappointed" grew by 1%. The ones who chose the latter option were asked to explain why, so here are the biggest reasons we observed how that could be:

@samuelsycamore Please advise on the copy here!

Reply:
Agree with @ryancogswell the current wording I see in https://deploy-preview-30999--material-ui.netlify.app/blog/2021-developer-survey-results/#how-would-you-feel-if-you-could-no-longer-use-mui is the opposite of what it should say:

> The "Somewhat disappointed" also saw a growth of 9%. These indicators suggest to us that we're on the right track with our products and their continued evolution.

It's not a growth of 9% of the somewhat disappointed. It's a reduction of 10.6% of the very disappointed (our target). These indicators suggests that we need to course correct, it's getting worse.